src/index.js

```diff
@@ -1,6 +1,9 @@
 import "./index.scss";
 import { initDopplerUI } from "./js";
+import { registerComponents, globalBehaviors } from "./components";
 
 if (window["style-guide-configuration"]?.autoInitialize) {
   initDopplerUI();
+  registerComponents();
```

Comment:
What do you think about running `registerComponents` always, whatever the value of `autoInitialize` is?